### DISCUSSION TITLE
Long lists : allow to jump immediately to a specific letter for Albums / Artists

### DISCUSSION THREAD

**Discussion by vingerha** [music-assistant/support#2632] [Feature requests and ideas]
I have over 100k of music pieces in 3500+ albums
When browsing through the list, it is a challenge if the Artist of Album is not in the first (say) 200 entries where browsing via the scrollbar to the end is slow esp. if the browser cache has been cleared for other reasons. 
It would be great to be able to jump to the starting letter of an Artist/Album
This selection mechanism should be outside of the scrolling section, suggestion is to make the header row frozen

**Comment by OzGav** [comment]
You can use the search

**Reply by vingerha** [reply]
Yes I know and I do agree this is not of high prio but the search is not taking the 'starting' letter, i.e. typing a 'c' will return hundreds of results still. Just trying to thnk how to make searching more friendly with large amounts of local stuff based on my experiences :)
Also, when scrolling, the search bar scrolls away with it... one could say this is just cosmetics but UX is key I believe (it is to me).

**Reply by OzGav** [reply]
Please add to the existing thread which I see you have discovered. https://github.com/orgs/music-assistant/discussions/2554

**Comment by vingerha** [comment]
Hmm... that is not very practical as this topic does not cover my request, imo adding different requests under the wrong header will make a mess out of it, your the admin but my 2cts from day-2-day work with dev-operations

**Reply by OzGav** [reply]
You said 

> I just created a new request with this very same demand.

thus the other thread contains the same as this

**Reply by vingerha** [reply]
I responded to the reply of someone else as this already deviated from the main post... with the exact smae intent, not mix various enhancement requests (or reduce to mix at a minimum)

**Reply by OzGav** [reply]
We get the idea
